components/fraudlabs_pro/actions/feedback-order/feedback-order.mjs

```diff
@@ -0,0 +1,52 @@
+import fraudlabsProApp from "../../fraudlabs_pro.app.mjs";
+
+export default {
+  name: "Feedback Order",
+  description: "Update status of a transaction from pending-manual-review to APPROVE, REJECT or IGNORE. The FraudLabs Pro algorithm will improve the formula in determine the FraudLabs Pro score using the data collected. Please refer to the [documentation](https://www.fraudlabspro.com/developer/api/feedback-order) for the explanation of the result returned.",
+  key: "fraudlabs_pro-feedback-order",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    fraudlabsProApp,
+    id: {
+      type: "string",
+      label: "ID",
+      description: "Unique transaction Id generated by Fraud Check API.",
+    },
+    format: {
+      propDefinition: [
+        fraudlabsProApp,
+        "format",
+      ],
+    },
+    action: {
+      type: "string",
+      label: "action",
+      description: "Perform APPROVE, REJECT, or REJECT_BLACKLIST action to transaction. Available values are: `APPORVE` which will approve a transaction, `REJECT` which will reject a transaction, and `REJECT_BLACKLIST` which will reject and blacklist the transaction. When a transaction was blacklisted, the information will be recorded into the system for future blacklist validation check.",
+    },
+    note: {
+      type: "string",
+      label: "note",
+      description: "*(optional)* Notes for the feedback request.",
```

Comment:
FYI the v2 UI currently groups all optional props in a clearly labeled "Optional" section, so it may look redundant to users to include `*(optional)*` for each optional prop.

Reply:
It's weird that I didn't see any "Optional" section, is there any additional config I can do?

Reply:
Ah you are likely using the legacy version of our workflow builder. If you create a "v2" workflow (top right of your main account page at pipedream.com), you should see this format, and you can see that it may appear overwhelming to a user:
![GIF 2022-03-03 at 7 20 59 PM](https://user-images.githubusercontent.com/20780753/156693081-1e879d28-e15b-43a6-bcac-9b432e8e1173.gif)

Reply:
Oh I see. Is there an option to do like in this screenshot?
> Hi @dannyroosevelt, ![screenshot-pipedream com-2022 03 04-10_56_52](https://user-images.githubusercontent.com/19896915/156694283-c6e5d1b0-0a54-4de1-bf60-375ada3f6633.png) Is there anyway to archive similar like this? Should be suitable to hide those optional options without removing it.

Reply:
Unfortunately not -- that is the legacy version of our actions in the legacy builder. I strongly recommend limiting the number of optional params to those that are most useful for your users.

Reply:
Hi @dannyroosevelt , I understood the problem here. Before we can decide on remove which option parameters, can you tell us is there a recommended number of optional parameters to be appear? Like 9 or 10.

Reply:
We typically recommend no more than 3-5 optional props.

Reply:
Hi @dannyroosevelt , thanks for the reply! Unfortunately, the number is too few for us to fit in the optional parameters that what we think is important, so we had took out the actions related. We will review this and consider to add it back in future though.